lib/Model/IMAPMessage.php

```diff
@@ -155,6 +155,10 @@ public function setFlags(array $flags) {
 		throw new Exception('Not implemented');
 	}
 
+	public function getRawMessage() {
+		return $this->fetch->getFullMsg();
```

Comment:
not used, please remove this method

Reply:
It is used, I'm sure.

Reply:
found it :) https://github.com/nextcloud/mail/pull/2264/files#r395078054